study.md

```diff
@@ -67,5 +68,5 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+The request goes to the router, then it gets matched to a controller and the appropriate action for the request. The action is carried out by the model. The action takes the request and passes it to the view.
```

Comment:
We won't be using Rails's views so once the models' actions are completed (if any), the controller will render a response to send back to the client.